docs/rest.md

```diff
@@ -311,8 +318,8 @@ See `sources` for `/2.0/snaps`.
 
 ### POST
 
-* Description: Install, update, remove, purge, activate, deactivate, or
-  rollback the snap
+* Description: Purchase, install, update, remove, purge, activate, deactivate,
```

Comment:
s/purchase/buy/, and please move the action to the end. We don't want to emphasize it, looking like every snap needs to be bought.